documentation/src/docs/security/pravega-security-authorization-authentication.md

```diff
@@ -0,0 +1,21 @@
+<!--
+Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+# TLS, Authorization, Authentication - Enabling encryption, authorization and authentication features
+Pravega stores the critical data for the customers.The goal is to run Pravega as a central streaming storage for different parts of the organizations having different levels of access to the data. This feature is crucial to supprt multi tenancy.
```

Comment:
Let's replace the first paragraph with the following:

```
Pravega ingests application data, which is often sensitive and requires security
mechanisms to avoid unauthorized access. To prevent such unauthorized accesses
in shared environments, we have enabled mechanisms in Pravega that secure
stream data stored in a Pravega cluster.  The security documentation covers
aspects of our mechanisms and explains how to configure Pravega to enable security.
```

Reply:
@arvindkandhare 
[Pravega controller](../security/pravega-security-configurations.md#Pravega controller) and [Pravega Segment store](../security/pravega-security-configurations.md#Pravega Segmentstore). These parameters can be changed to represent required security configurations.
Please resolve this